kuma/settings/testing.py

```diff
@@ -85,3 +85,6 @@
 
 # Use a dedicated minio bucket for tests
 ATTACHMENTS_AWS_STORAGE_BUCKET_NAME = 'test'
+
+# Never enabled in tests.
+SENTRY_DSN = None
```

Comment:
This was a total timesink. I had put the `mdn-stage` Sentry DSN into my personal `.env` and it broke all sorts of tests when running `pytest`.